stream/src/js/stream.js

```diff
@@ -54,7 +54,9 @@ $(document).ready(function() {
 
     $('#video').on('click', function() {
         /* TODO: Correct video URL */
-        $(this).html('<div class="video-background"><iframe title="Live Stream Video" src="https://www.youtube.com/embed/e2axToBYD68?autoplay=1" allowfullscreen></iframe></div>');
+        $(this).html('<iframe title="Live Stream Video" src="https://www.youtube.com/embed/e2axToBYD68?autoplay=1" allowfullscreen></iframe>');
+
+        $('.hero').addClass('video-playing');
```

Comment:
The semi transparent background needs to be applied to the full width container, which in this new configuration is `.hero`